### PR TITLE
Add yangpa as Kubeflow member

### PR DESCRIPTION
**Pull request by IronPan**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/193)

### CODE REVIEW

File: members.yaml [hunk 1]
```diff
@@ -135,6 +135,13 @@
   lastName: Garg
   affiliation: Cisco
   email: anubhgar@cisco.com
+
+- github: ironpan
```
Comment (gaocegege) [inline members.yaml]: Please put your name after inc0 since the file is in alphabetical order

Comment (IronPan) [reply members.yaml]: done

**Pull request merged**